en/tutorial-base.md

```diff
@@ -536,15 +603,20 @@ class SignupController extends Controller
     }
 }
 ```
+<br/>
 
-We then instantiate the Users class, which corresponds to a User record. The class public properties map to the fields of the record in the users table. Setting the relevant values in the new record and calling `save()` will store the data in the database for that record. The `save()` method returns a boolean value which indicates whether the storing of the data was successful or not.
+At the beginning of the **registerAction** we create an empty user object from the Users class, which manages a User's record. The class's public properties map to the fields of the users table in our database. Setting the relevant values in the new record and calling `save()` will store the data in the database for that record. The `save()` method returns a boolean value which indicates whether the storing of the data was successful or not.
```

Comment:
OK so long as we are consistent everywhere. Could you just add a note of this in the README file?

Other than that it is good to go.